src/gui/core/plugin/userui/src/main/java/org/geoserver/geofence/gui/server/service/impl/RulesManagerServiceImpl.java

```diff
@@ -53,11 +58,6 @@
 import com.vividsolutions.jts.geom.Polygon;
 import com.vividsolutions.jts.io.ParseException;
 import com.vividsolutions.jts.io.WKTReader;
-import org.geoserver.geofence.core.model.IPAddressRange;
-import org.geoserver.geofence.core.model.enums.CatalogMode;
-import org.geoserver.geofence.gui.client.model.data.ClientCatalogMode;
-import org.geoserver.geofence.services.util.IPUtils;
-import org.springframework.dao.DuplicateKeyException;
```

Comment:
Please avoid committing packages cleanup if this is the only change inside a class